tests/yam/verify_yast2_control_center.pm

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 SUSE LLC
+#
+# Summary: check that YaST2 control center with both Qt
+#          and Ncurses interfaces still work after the updates.
+#
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'y2_module_guitest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
```

Comment:
we have a library that launch YaST in the same fashion for ncurses or qt (I think I mentioned to you offline), so we don't need different code to handle this.

Reply:
I couldn't find it for opening the control-center. The code seems to be only for opening module, but maybe I'm looking in the wrong place. Could you please point me to the right code? thx